### PR TITLE
guides/integrations/redis: use updated Jedis integration

### LINKED ISSUE #474: integrations/redis: update the Java integrations guide
The current section with Jedis on Java is a little painful to use as it requires folks to make a fork.
The reason being that we had no way to publish and the official OpenCensus Java groupId is on lockdown and requires some approval but also means that the core OpenCensus Java team has to maintain those integrations.

Anyways I spared some time over the past 3 weeks and made a Jedis integration that wraps the Jedis client and extends Jedis hence non-invasive but also published on Maven central 

Resource|URL
---|---
Github URL|https://github.com/orijtech/ocjedis
Maven Central|https://mvnrepository.com/artifact/io.orijtech.integrations/ocjedis/0.0.2

which can be imported simply
```xml
<!-- https://mvnrepository.com/artifact/io.orijtech.integrations/ocjedis -->
<dependency>
    <groupId>io.orijtech.integrations</groupId>
    <artifactId>ocjedis</artifactId>
    <version>0.0.2</version>
</dependency>
```

and after that import, just the OpenCensus imports and usable normally without any further change

and here is a fully functioning itunes search app https://github.com/orijtech/itunes-search/blob/c0066aaa8cbef6106354384585771d44592b0ba1/src/main/java/io/mediasearch/search/MediasearchClient.java#L48-L68

### PR DESCRIPTION
**Pull request by odeke-em**
The Jedis integration is now available at
https://mvnrepository.com/artifact/io.orijtech.integrations/ocjedis

and it is published under my domain because:
a) publishing under io.opencensus.integrations is a complex issue
that requires multi-level permissions and approvals but also
b) means that the OpenCensus Java team is obligated to maintain
the integration forever

Being able to update artifacts makes developing integrations a lot
smoother but also releases the small team from obligations.
The code was written and released by Orijtech, Inc. and provides
an Apache License on behalf of OpenCensus Authors.

Fixes #474

### CODE REVIEW

File: content/guides/integrations/redis/Java.md [hunk 1]
```diff
@@ -9,152 +9,167 @@ logo: /images/java-opencensus.png
...
-import io.opencensus.exporter.stats.stackdriver.StackdriverStatsConfiguration;
-import io.opencensus.exporter.stats.stackdriver.StackdriverStatsExporter;
-import io.opencensus.exporter.trace.stackdriver.StackdriverTraceConfiguration;
-import io.opencensus.exporter.trace.stackdriver.StackdriverTraceExporter;
+import io.opencensus.trace.Tracer;
 import io.opencensus.trace.Tracing;
+import io.opencensus.trace.Span;
+import io.opencensus.trace.Status;
 import io.opencensus.trace.config.TraceConfig;
 import io.opencensus.trace.config.TraceParams;
```
Comment (mayurkale22) [inline content/guides/integrations/redis/Java.md]: Unused import, please remove it.

File: content/guides/integrations/redis/Java.md [hunk 2]
```diff
@@ -9,152 +9,167 @@ logo: /images/java-opencensus.png
...
+        } catch(Exception e) {
+            System.err.println("Failed to create OpenCensus exporters " + e);
+        }
 
         while (true) {
             try {
-                System.out.print("> ");
-                System.out.flush();
-                String query = stdin.readLine();
+                Thread.sleep(1700);
```
Comment (mayurkale22) [inline content/guides/integrations/redis/Java.md]: Can you please add a comment here about 1700 sleep?

File: content/guides/integrations/redis/Java.md [hunk 3]
```diff
@@ -9,152 +9,167 @@ logo: /images/java-opencensus.png
...
-            StackdriverStatsExporter.createAndRegister(
-                StackdriverStatsConfiguration.builder()
-                .setProjectId(gcpProjectId)
-                .build());
-        } catch (Exception e) {
-            System.err.println("Failed to setup OpenCensus " + e);
-        }
+    private static void setupOpenCensusAndExporters() throws Exception {
+        // Enable exporting of all the Jedis specific metrics.
+        Observability.registerAllViews();
```
Comment (mayurkale22) [inline content/guides/integrations/redis/Java.md]: We are calling ```Observability.registerAllViews()``` twice, any reason?

File: content/guides/integrations/redis/Java.md [hunk 4]
```diff
@@ -178,35 +193,40 @@ public class JedisOpenCensus {
...
-            <artifactId>opencensus-exporter-stats-stackdriver</artifactId>
+            <artifactId>opencensus-api</artifactId>
             <version>${opencensus.version}</version>
         </dependency>
 
         <dependency>
             <groupId>io.opencensus</groupId>
-            <artifactId>opencensus-api</artifactId>
+            <artifactId>opencensus-impl</artifactId>
             <version>${opencensus.version}</version>
```
Comment (mayurkale22) [inline content/guides/integrations/redis/Java.md]: ```<scope>runtime</scope>``` ?

File: content/guides/integrations/redis/Java.md (hunk 1 above)
Comment (odeke-em) [reply content/guides/integrations/redis/Java.md]: We use these two imports in the code, please take a look https://raw.githubusercontent.com/census-instrumentation/opencensus-website/b41039e38360fac7d6fc7d71a8cacdee09a89088/content/guides/integrations/redis/Java.md

File: content/guides/integrations/redis/Java.md (hunk 3 above)
Comment (odeke-em) [reply content/guides/integrations/redis/Java.md]: My mistake, I'll fix that.

File: content/guides/integrations/redis/Java.md (hunk 2 above)
Comment (odeke-em) [reply content/guides/integrations/redis/Java.md]: Roger that, thanks!

File: content/guides/integrations/redis/Java.md (hunk 4 above)
Comment (odeke-em) [reply content/guides/integrations/redis/Java.md]: Roger that, I'll make that update.

File: content/guides/integrations/redis/Java.md (hunk 1 above)
Comment (odeke-em) [reply content/guides/integrations/redis/Java.md]: Thanks for the offline clarification that you meant TraceParams, I'll remove that.

**Comment by mayurkale22** [comment]
Looks Good. Thank you

**Comment by odeke-em** [comment]
Thank you @mayurkale22 for the review!

**Pull request merged**